src/harness.js

```diff
@@ -205,13 +205,14 @@ class Harness {
   }
 
   /**
-   * Loads and fills a contact form,
+   * Loads and fills a contact form with the appropriate action
    *
    * @param {string} contactType Type of contact that should be created
+   * @param {string} action one of 'create' or 'edit'
    * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
    */
-  async fillContactForm(contactType, ...answers) {
-    const xformFilePath = path.resolve(this.options.contactXFormFolderPath, `${contactType}-create.xml`);
+  async _fillContactForm(contactType, action, ...answers) {
```

Comment:
Consider moving this outside the class since it's technically not a public interface